src/metadata.mjs

```diff
@@ -142,6 +142,7 @@ const createMetadata = slugger => {
         api: apiDoc.stem,
         slug: sectionSlug,
         source_link,
+        api_doc_source: apiDoc.basename,
```

Comment:
Should we have this be the path relative to the nodejs root? I.e. `doc/api/buffer.md` instead of it `buffer.md`?

Reply:
yes, relative path would be good :)